_rules/6cbcbe.md

```diff
@@ -0,0 +1,279 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names resolve to the same resource or equivalent resources.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names](#accessible-name) that does not only consist of [whitespace](#whitespace).
+
+**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name](#accessible-name).
+
+### Expectation
+
+When followed, the links in each set of target elements resolve to the [same resource](#same-resource) or to [equivalent resources](#equivalent-purpose).
+Resolving the links includes potential redirects, if the redirects happen instantly.
+
+## Assumptions
+
+This rule assumes that the purpose of the links with identical names would not be ambiguous to users in general when seen in context on the web page, as users of assistive technologies in this case is not at a disadvantage when viewing the link out of context, e.g. on a list of links in a screen reader.
```

Comment:
The second part of the sentence is not easy to read and understand. Since I'm not sure about what you meant, I'm not able to offer a suggestion :-(

Reply:
"as users of assistive technologies in this case is not at a disadvantage when viewing the link out of context, e.g. on a list of links in a screen reader."

 should be (I think)

". This assumption is made because, in the case described here, users of assistive technologies are not at a disadvantage when viewing the link out of context, e.g. on a list of links in a screen reader"

However that makes little sense because if they are not ambiguous when seen in context that doesn't mean that they are clear without context right?

Reply:
From the Understanding document: 

> The Success Criterion includes an exception for links for which the purpose of the link cannot be determined from the information on the Web page. In this situation, the person with the disability is not at a disadvantage; there is no additional context available to understand the link purpose. However, whatever amount of context is available on the Web page that can be used to interpret the purpose of the link must be made available in the link text to satisfy the Success Criterion.

(https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-link.html)

I will have another look at the wording asap.

Reply:
@ShadowBB 
> However that makes little sense because if they are not ambiguous when seen in context that doesn't mean that they are clear without context right?

I think the point here is that if I created a page with nothing but a link that reads "Click here", it is not an accessibility issue, and thus not a violation of success criterion 2.4.9, because it gives the same (poor) experience to everyone.
And I guess it doesn't just have to be a poor experience, but could also be game-like or artsy.

Reply:
@carlosapaduarte and @ShadowBB, I gave a shot at a rewrite. Please have a look again.

Reply:
Oh! I totally interpreted that the other way around. 

You new writeup makes this a lot more clear and I agree with it wholeheartedly now! Nice work!